### ISSUE TITLE
[Feature] Make dashboard path configurable

### ISSUE THREAD

**Issue by limmesberger** [jobrunr/jobrunr#171]
I would like to have multiple dashboards available through a reverse proxy via different paths. For this it would be great to be able to configure the path to the dashboard.

e.g.:
http://service-a:8000/service-a/dashboard -> http://reverse-proxy/service-a/dashboard
http://service-b:8000/service-b/dashboard -> http://reverse-proxy/service-b/dashboard

**Comment by rdehuyss** [comment]
I assume this is for the all the links within the dashboard?

**Comment by limmesberger** [comment]
That would be great, if the configured base-path will be used for all links within the dashboard.

**Comment by rdehuyss** [comment]
I investigated this and I think it is a rather complex change:
- change all react files to support configurable path
- API calls also needs to be configurable then (and all calls to it)

Currently, I don't know if it is worth the effort to be honest. Let me rephrase it - would you consider a Pro license if it was part of the Pro package :smile:  :wink: ?

Off-course, I'm open for a PR for this change but I think testing to make sure all stuff still works correctly, will also take some time.
